meson.build

```diff
@@ -4,6 +4,12 @@ project('libratbag', 'c',
 	default_options : [ 'c_std=gnu99', 'warning_level=2' ],
 	meson_version : '>= 0.40.0')
 
+# The DBus API version. Increase this every time the DBus API changes.
+# No backwards/forwards guarantee, clients are expected to understand
+# whatever ratbagd speaks or bail out. This should be removed if we ever
+# finish the API and declare it stable.
+ratbagd_api_version = 1
```

Comment:
Are we forced to use an int here?

I'd prefer have a string like `temp_version_42` (or with some random words) to ensure the fact that this is not compatible across revisions?

Or maybe start at `42` and say we will *semi-randomly* pick up the next number by using the `rand()` function with `42` as a seed.

Reply:
we can use whatever we want, but 
- integers are easiest to parse, send across, etc.
- integers are easiest to explain, they just go up. and up. and up and up. the conversation "look, you have version "'goblet' but you need to update to version 'wineglass'" is something I'd rather avoid
- you're trying to protect against a theoretical mass influx of people developing ratbagd's API and using it from other clients by trying to randomize this. Realistically, it's going to be a grand total of maybe 3 developers. And as long as `n != n+1` is true, the numbers are just fine.